### DISCUSSION TITLE
Font availability on Linux?

### DISCUSSION THREAD

**Discussion by gwerbin** [Homebrew/discussions#877] [Linux]
Would it be somehow possible to install fonts under Linux? Last I tried, I was unable to do it because they are Casks, which are not supported on Linux.

However it' a shame that fonts specifically are Mac-only as a result of this situation, when for the most part they are platform-agnostic TTF and OTF files.

Is it possible to at least work around this by writing a Ruby script? Maybe something that generates a Formula from a Cask, or something that extracts URLs from Casks and manually fetches them.

I tried to do this myself at one point, but I couldn't figure out a way to load/instantiate a Cask from the Ruby API (using `brew irb`/`brew ruby`). Any ideas?

**Comment by iMichka** [comment]
I think there are some fonts in https://github.com/Linuxbrew/homebrew-xorg
There is also this project: https://github.com/Linuxbrew/homebrew-fonts

**Reply by gwerbin** [reply]
> I think there are some fonts in https://github.com/Linuxbrew/homebrew-xorg
> There is also this project: https://github.com/Linuxbrew/homebrew-fonts

Thanks, the Xorg fonts are "old style" X11 fonts and not modern TTF/OTF fonts. However it looks like the 2nd repo you linked does the job. I `tap`-ed it and now I'm able to get all the fonts I wanted. Many of them appear to be installed as "HEAD only", but that's fine for my purposes. Thank you!

Edit: maybe this (and `homebrew-xorg`) should be added to the [Linuxbrew docs](https://docs.brew.sh/Homebrew-on-Linux)? I think it could increase the general attractiveness of Homebrew on Linux if more people knew about it.
